packages/aggregator/src/tests/analytics/aggregateAnalytics/aggregations/sumAcrossPeriods.test.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+import { sumAcrossPeriods } from '../../../../analytics/aggregateAnalytics/aggregations/sumAcrossPeriods';
+
+describe('sumAcrossPeriods', () => {
+  const arrayToAnalytics = arrayAnalytics =>
+    arrayAnalytics.map(([dataElement, organisationUnit, period, value]) => ({
+      dataElement,
+      organisationUnit,
+      period,
+      value,
+    }));
+
+  const ANALYTICS = arrayToAnalytics([
+    // BCD01 - TO
+    ['BCD01', 'TO', '201601', 1],
+    ['BCD01', 'TO', '201801', 0.1],
+    ['BCD01', 'TO', '201901', 0.01],
+    // BCD01 - PG
+    ['BCD01', 'PG', '201601', 2],
+    ['BCD01', 'PG', '201801', 0.2],
+    ['BCD01', 'PG', '201901', 0.02],
+    // BCD02 - TO
+    ['BCD02', 'TO', '201801', 4],
+    ['BCD02', 'TO', '202001', 0.44],
+  ]);
```

Comment:
This is a new analytics definition style (`arrayAnalytics`). It's not used in actual code but I find it very convenient for tests.

Reply:
I've seen you use it before in the indicator package I think. Very clean!